### DISCUSSION TITLE
License clarification

### DISCUSSION THREAD

**Discussion by tschbc** [SixLabors/ImageSharp#2784] [Q&A] [answered]
@JimBobSquarePants I'd like to add ImageSharp to the project I'm currently working on:

- **Project is licensed with Apache 2.0** ✔
- Government organization ⚠
- The repository is currently in GitHub Enterprise, so not publicly accessible ⚠

Based on the license, I _think_ the Apache 2.0 license applies to the project, but I get nervous when interpreting licenses and my brain starts failing.

Does this track or am I way off base?

**Comment by JimBobSquarePants** [comment]
Will the source code be available? The whole point of OSI licensed software is that it must be.

**Reply by tschbc** [reply]
I want to move it to our public org Soon™, but have no definitive plans yet.

**Reply by JimBobSquarePants** [reply, marked as answer]
Cool. Yeah, the Apache license stands then. You don't require a commercial license.

**Reply by tschbc** [reply]
Thank you for the quick answer.
